packages/desktop-client/src/components/tooltips.tsx

```diff
@@ -123,11 +169,11 @@ export class Tooltip extends Component {
 
     if (
       container.parentNode &&
-      container.parentNode.style.overflow === 'auto'
+      (container.parentNode as HTMLElement).style.overflow === 'auto'
     ) {
-      return container.parentNode;
+      return container.parentNode as HTMLElement;
     }
-    return container;
+    return container as HTMLElement;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏anything we could do to get rid of all these manual type coercions too? Maybe specifically define `getContainer(): HTMLElement` return type or another solution.